flang/test/Semantics/call10.f90

```diff
@@ -109,10 +109,6 @@ pure subroutine s05 ! C1589
     !ERROR: A pure subprogram may not initialize a variable
     real :: v3
     data v3/0./
-    !ERROR: A pure subprogram may not have a variable with the SAVE attribute
-    real :: v4
-    common /blk/ v4
```

Comment:
If you leave these two lines, you'll test to ensure that we don't emit an error message for common block declarations in pure procedures.

Reply:
Done.